shared/provision/username-or-email/index.tsx

```diff
@@ -141,15 +141,7 @@ const styles = Styles.styleSheetCreate(
         isMobile: {...Styles.padding(Styles.globalMargins.small)},
       }),
       wrapper: Styles.platformStyles({
```

Comment:
merge patricks pr again

Reply:
yeah - i just got rid of those commits and changed the base to his branch. i'll move it to master once his gets merged.